addons/sale_management/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api, _
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.onchange('property_product_pricelist')
+    def _onchange_pricelist(self):
```

Comment:
```suggestion
    def _onchange_property_product_pricelist(self):
```

Let's respect the [Odoo code Guidelines](https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#symbols-and-conventions): "Onchange method : the onchange method pattern is `_onchange_<field_name>`"